slatedb-bencher/src/args.rs

```diff
@@ -232,6 +252,55 @@ impl Display for KeyGeneratorType {
     }
 }
 
+#[derive(Clone, Copy)]
+pub(crate) enum IsolationLevelType {
```

Comment:
Can you get rid of this and use `IsolationLevel` in `transaction_manager.rs` directly instead?